spec/features/breadcrumbs_spec.rb

```diff
@@ -48,8 +48,8 @@
 
     it "includes the current page in plain text" do
       page.within ".breadcrumbs" do
-        is_expected.not_to have_link "Train to be a teacher"
-        is_expected.to have_text "Train to be a teacher"
+        is_expected.not_to have_link "Home Train to be a teacher"
```

Comment:
@plisovin Just want to check what these changes do and why we need them here? Thanks!

Reply:
It was test failing previously, and then something has changed again, so I reverted this test in this [commit](https://github.com/DFE-Digital/get-into-teaching-app/pull/3465/commits/592c42275e970474030e2b79ed4fe6c46f48d6ec)